[WIP] feat(accounts-orgs-docs): add initial docs related to accounts and organizations (#406)
localstack/docs: content/en/user-guide/web-application/organizations-teams/index.md

---
title: "Organizations / Teams"
tags: [] 
weight: 2
description: >
  Organizations / Teams offer a flexible way of managing your needs within the LocalStack domain.
---

### Organizations / Teams
<i>The terms 'Organization' and 'Team' are used interchangeably within our web application</i><br>
<br>
Both individual and company accounts are able to invite other accounts into their teams.<br>
Accounts that are invited into a team can also be given advanced role settings by the team admin, 
such as purchasing subscriptions, managing payment methods, and more.

> <br>Teams offer an easy way to manage subscriptions and api keys for multiple people.

Reviewer: ```suggestion
<br>Teams offer an easy way to manage subscriptions and API keys for multiple individual users.
```

> For example you can purchase a plan with multiple seats for your team, which allows you to distribute the API keys among your team members.

Reviewer: ```suggestion
For example, you can purchase a plan with multiple seats for your team, which allows you to distribute the API keys among your team members.
```

#### Inviting Team Members
To invite team members, navigate to the [`Team Members`](https://app.localstack.cloud/account/members) section of your profile.<br>

> Enter the user name and the email address of an team member, and click on the plus symbol to invite them to your team.

Reviewer: ```suggestion
Enter the user name and the email address of a team member, and click on the plus symbol (`+`) to invite them to your team.
```
